### PR TITLE
Mehrdad lint

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -39,10 +39,11 @@ jobs:
         pip install -r ./collect_metadata/requirements.txt
         pip install -r ./short_term/requirements.txt
         pip install -r ./long_term/requirements.txt
+        pip install -r ./dashboard/requirements.txt
```
Comment (Ruyzambrano) [inline .github/workflows/main.yml]: I feel there must be an easier way to do this other than installing each file manually...

File: dashboard/longterm_extract.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ def download_truck_data_files(s3_client, target_bucket) -> list[str]:
 
 
 def extract_s3_files() -> list[str]:
-    """"""
+    """The main function."""
```
Comment (Ruyzambrano) [inline dashboard/longterm_extract.py]: You might be more specific here. Yes it is the main function, but what does it do/return?

File: dashboard/longterm_transform.py [hunk 3]
```diff
@@ -37,6 +37,7 @@ def get_times_watered(df: pd.DataFrame) -> alt.Chart:
 
 
 def get_max_temp_per_plant(df: pd.DataFrame) -> alt.Chart:
+    """Creates a chart of max temperature for every plant."""
```
Comment (Ruyzambrano) [inline dashboard/longterm_transform.py]:
```suggestion
    """Returns a chart of max temperature for every plant."""
```

File: dashboard/longterm_transform.py [hunk 4]
```diff
@@ -52,6 +53,7 @@ def get_max_temp_per_plant(df: pd.DataFrame) -> alt.Chart:
 
 
 def get_min_moisture_per_plant(df: pd.DataFrame) -> alt.Chart:
+    """Creates a chart of min moisture for every plant."""
```
Comment (Ruyzambrano) [inline dashboard/longterm_transform.py]:
```suggestion
    """Returns a chart of min moisture for every plant."""
```

File: short_term/.coverage
Comment (Ruyzambrano) [inline short_term/.coverage:1]: Seems like your `.gitignore` didn't work here

**Review by Leon-JS02 — APPROVED** [review]
Good job on restructuring the workflow requirements.

File: dashboard/load_dashboard.py [hunk 5]
```diff
@@ -6,8 +6,8 @@
 from transform_dashboard import create_botanist_pie, create_temperature_bar
 from transform_dashboard import create_moisture_bar, create_temperature_line
 from longterm_extract import extract_s3_files
-from longterm_transform import concat_csvs, get_times_watered, get_max_temp_per_plant, get_min_moisture_per_plant
-
+from longterm_transform import concat_csvs, get_times_watered
+from longterm_transform import get_max_temp_per_plant, get_min_moisture_per_plant
```
Comment (YakMan101) [inline dashboard/load_dashboard.py]: Instead you can use:
```
from longterm_transform import (concat_csvs, get_times_watered,
get_max_temp_per_plant, get_min_moisture_per_plant)
```

File: dashboard/longterm_extract.py [hunk 6]
```diff
@@ -48,7 +48,8 @@ def download_truck_data_files(s3_client, target_bucket) -> list[str]:
 
 
 def extract_s3_files() -> list[str]:
-    """"""
+    """The main function.
+        Extracts long term files from the S3 bucket."""
```
Comment (YakMan101) [inline dashboard/longterm_extract.py:52]: Thanks for adding extra detail

**Review by YakMan101 — CHANGES_REQUESTED** [review]

**Review by YakMan101 — APPROVED** [review]

**Pull request merged**